test/files/rules/varname.test.ts

```diff
@@ -30,3 +30,5 @@ export function anotherFunctionWithInvalidParamNames ([first_element, SecondElem
 export function functionWithInvalidSpread(invalid_arg: ...number) { // 1 failure
   //
 }
+
+let optionallyValid_ = "bar";
```

Comment:
can you please add a test that verifies behavior when _both_ options (leading & trailing) are enabled?